examples/sid_visualization/commands.sh

```diff
@@ -0,0 +1,53 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# To enable
+export DOCKER_BUILDKIT=1
+export COMPOSE_DOCKER_CLI_BUILD=1
+
+export MORPHEUS_HOME=$(git rev-parse --show-toplevel)
+
+# Change to morpheus home
+cd ${MORPHEUS_HOME}
+
+export DOCKER_IMAGE_TAG="sid-viz"
+
+# Build the dev container
+./docker/build_container_dev.sh
+
+# Change to the example directory
+cd ${MORPHEUS_HOME}/examples/sid_visualization
+
+# Launch the containers
+docker-compose up -d
+
+# Exec into the morpheus container
+docker-compose exec morpheus bash
+
+# Inside the container, compile morpheus
+BUILD_DIR=build-docker ./scripts/compile.sh
+
+# Install morpheus with an extra dependency
+pip install -e . && pip install websockets
+
+# Now run the demo
+python examples/sid_visualization/run.py \
+  --use_cpp=False \
+  --num_threads=1 \
+  --triton_server_url=triton:8001 \
+  --input_file=./examples/data/sid_visualization/group1-benign-2nodes-v2.jsonlines \
+  --input_file=./examples/data/sid_visualization/group2-benign-50nodes.jsonlines \
+  --input_file=./examples/data/sid_visualization/group3-si-50nodes.jsonlines \
+  --input_file=./examples/data/sid_visualization/group4-benign-49nodes.jsonlines
```

Comment:
Is this just a holdover from my branch? Not sure we want these raw commands. If they are just from that branch, can you make sure that they all have been incorporated into the readme somewhere? And then delete this file